### ISSUE TITLE
Problem generating commit messages on Windows

### ISSUE THREAD

**Issue by wbecher** [alufers/commit-o-matic#1]
Hi!

I'm using VSCode on Windows 11. I have git installed on my system, and I'm able to use git command line on any terminal (cmd, powershell, git bash), but when I click the commit-o-matic button to generate the message, nothing happens.

In the extension logs, I got this error:

![image](https://user-images.githubusercontent.com/1504625/227616476-e7f4da7a-9108-46bd-ae22-91e23dc968d0.png)

```
2023-03-24 15:53:20.335 [error] Error: spawn git ENOENT
    at Process.ChildProcess._handle.onexit (node:internal/child_process:283:19)
    at onErrorNT (node:internal/child_process:478:16)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
```

Below are some screenshots I took, running `git diff --staged` on windows cmd, powershell and git bash:

Windows CMD
![image](https://user-images.githubusercontent.com/1504625/227615710-a42654dd-eb68-4d2f-97bd-3a63886f7dd9.png)

Powershell
![image](https://user-images.githubusercontent.com/1504625/227615835-e7a5c9e8-5975-4810-816a-559b8485e020.png)

Git Bash
![image](https://user-images.githubusercontent.com/1504625/227615954-b7485814-ba2f-4a34-972f-fc03ad4af778.png)

**Comment by alufers** [comment]
Huh, weird.

I will have to try on windows myself. Probably it somehow cannot find git in the path

**Comment by wbecher** [comment]
Yeah, it´s very weird... as you can see I have the git folder in the PATH. If you need me to run some tests on my machine, don't mind to ask.
